frameworks.rst

```diff
@@ -93,11 +93,11 @@ links!
 `weblayer <http://packages.python.org/weblayer>`_
     weblayer is a lightweight, componentised package for writing WSGI
     applications.
-`Zope 3 <http://www.zope.org/>`_
-    The venerable Python web framework, recreated anew in Zope 3, and
-    now a WSGI application.  It *seems* to have some WSGI bits deep
-    inside the publisher, but they aren't really documented at this
-    time.
+`Zope <https://zope.readthedocs.io>`_
```

Comment:
```suggestion
- `Zope <https://zope.readthedocs.io>`_
+ `Zope 4 and above <https://zope.readthedocs.io>`_
```

Reply:
i find that putting just «zope» is confusing. There is now even zope 4 and 5.

Reply:
"4" and "5" are just version numbers of the same product, Zope. There is no reason to distinguish.